vulcan/lib/server/workflows/umap.cwl

```diff
@@ -88,6 +88,11 @@ inputs:
     label: 'Number of iterations?'
     default: 0
     doc: 'The number of iterations for optimization - by default (0) either 200 for small datasets or 500 for large ones.'
+  6_Cluster_Differential_Expression__ignore_prefixes:
+    type: string
+    default: 'MT-,RPL,RPS'
+    label: 'Gene prefixes to ignore'
+    doc: 'A set of strings, separated by commas, for which gene symbols starting with these strings should NOT be shown in the umap overlay. Not case-sensitive, so the same strings will work for both human and mouse ribo & mito genes. Note: this does not affect the full differential expression table that can be downloaded.'
```

Comment:
@coleshaw I noticed that this input does still get shown right from the beginning even though I did not add this primary input as a "mock" input to the first cell.  Perhaps that bug is fixed and we can remove that long list of unnecessary inputs from the first cell?